pom.xml

```diff
@@ -31,11 +31,11 @@
 
     <properties>
         <connect-runtime-version>2.0.0</connect-runtime-version>
-        <confluent.version>5.0.0</confluent.version>
+        <confluent.version>5.1.0</confluent.version>
         <confluent.avro.generator.version>0.2.0</confluent.avro.generator.version>
         <junit.version>4.12</junit.version>
         <avro.version>1.8.2</avro.version>
```

Comment:
looks common is using 1.8.1 https://github.com/confluentinc/common/blob/master/pom.xml#L49

Reply:
Addressed in #15